examples/python/route_guide/run_codegen.py

```diff
@@ -28,5 +27,16 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Runs the protoc with gRPC plugin to generate protocol messages and gRPC stubs.
-python -m grpc.tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/route_guide.proto
```

Comment:
What is the motivation for this change?  My understanding was that python -m grpc.tools.protoc was the "blessed" way to build protos.  If we change this file, then there several references in the documentation to this file that need to be updated as well.